io/net/sriov_device_test.py.data/migratable_sriov_generic.yaml

```diff
@@ -0,0 +1,17 @@
+hmc_username: ""
+hmc_pwd: ""
+sriov_adapter: "U78D8.ND0.FGXX-P0-C2-C0"
+sriov_port: "0"
+ipaddr: ""
+netmasks: ""
+peer_ip: ""
+mac_id: ""
+migratable: 1
+backup_device_type: "veth"
+backup_veth_vnetwork: ""
+vnic_sriov_adapter: "U78D8.ND0.FGXX-P0-C2-C0"
```

Comment:
Remove  this param backup_device_type: "veth"  as I am making it local in code

Keep the value to `None` for below params
backup_veth_vnetwork: None
vnic_sriov_adapter: None

so I can handle in code to work for both vnic and veth